views/forms/ChartForm.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import Button from '@material-ui/core/Button'
+
+import { createChart } from '../fe-utils/fetchUtil'
+
+import Form from './Form'
+import BarChartFields from './BarChartFields'
+
+const ChartForm = ({ classes }) => {
+  const [formValues, setFormValues] = useState({})
+  const [fieldLabelValueMap, setFieldLabelValueMap] = useState([])
```

Comment:
should this be part of the `formValues` state?

Reply:
[78b1245](https://github.com/AMP-SCZ/dpdash/pull/84/commits/78b12458361f85aeb0919c885be4ca8c8f6c047b)